SIT.Manager/Controls/EmbeddedProcessWindow.cs

```diff
@@ -102,6 +102,11 @@ public async Task StartProcess()
                 // Set ProcessWindowHandle to the MainWindowHandle of the process
                 ProcessWindowHandle = _p.MainWindowHandle;
             }
+            else // linux
+            {
+                // Wait until the process has exited
+                await _p.WaitForExitAsync();
+            }
```

Comment:
This is some older stuff I added when trying to improve the installer, which I will finish at a later date.
I get an error because  [PatchViewModel.cs#L99](https://github.com/stayintarkov/SIT.Manager.Avalonia/blob/4238bf66ca0f75f86b4ed2cb7f7afa121996167d/SIT.Manager/ViewModels/Installation/PatchViewModel.cs#L99) only starts the process and doesn't wait for it to finish, but does then try to read the exit code. This was a fix for it but it should probably be in PatchViewModel instead

Reply:
You're both right.... I realised I made a small oopsie....

https://github.com/stayintarkov/SIT.Manager.Avalonia/blob/4238bf66ca0f75f86b4ed2cb7f7afa121996167d/SIT.Manager/ViewModels/Installation/PatchViewModel.cs#L99

The above was supposed to be ```await embeddedProcessWindow.WaitForExit();```

Probably what I deserve for not testing things properly on Linux 😓 


The whole patching thing is something to look into in the future cause it's a pain on Windows as well as Linux, although Linux is far worse.